src/views/organisationType.njk

```diff
@@ -1,5 +1,58 @@
 {% extends 'base.njk' %}
 {% set title = 'Organisation Type' %}
+{% from "govuk/components/radios/macro.njk" import govukRadios %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+{% block beforeContent %}
+
+    {{ govukBackLink({
+  text: "Back",
+  href: "/"
+}) }}
+{% endblock %}
 {% block content %}
-    {# placeholder for content #}
+
+    <form method="get" action="#">
+        {{ govukRadios({
+  name: "organisationType",
+  
+  fieldset: {
+    legend: {
+      text: "Organisation type",
+      isPageHeading: true,
+      classes: "govuk-fieldset__legend--l"
+      
+    }
+  },
+  items: [
+    {
+      value: "governmentDepartmentOrMinistry",
+      text: "Government department or Ministry"
+    },
+    {
+      value: "executiveAgency",
+      text: "Executive Agency"
+    },
+    {
+      value: "armsLengthBody",
+      text: "Arms length body"
+    },
+    {
+      value: "other",
+      text: "Other"
+    }
+  ],
+  errorMessage: {
```

Comment:
Already spoken on slack about error handling, but just making a note here for review purposes :) 

"we only want that appear if a user doesn’t select a button and clicks continue… if you take a look at an existing example on the IPV-DCA repo /src/views/journeys/prod/selectDevicePage.njk this has a good example"

Reply:
thank you , used as reference and added it